### DISCUSSION TITLE
Custom domain and SSL Certificate issues

### DISCUSSION THREAD

**Discussion by homogenic1000** [community/community#138468] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi,

English is not my primary language so sorry in advance

So I'm using Github pages for my website and I also use a custom domain that I buy from Hostpoint, and since I use my custom domain SSL Certificate don't work, so I used Zero SLL to generate a certificate and I register the certificate to my Hostpoint panel and I read documentation about third party SSL and the assistance of Hostpoint told me I should set up my SSL Certificate on the GitHub Pages panel.

I'm really lost and don't know what to do, so if you can help me that's cool :)

**Comment by meower1** [comment, marked as answer]
Hi, its actually a fairly simple process you just have to create a cname record and point it to the github pages link provided to you which is often in the following format: `<username>.github.io` and then in the pages section of your website's github repository add the subdomain you've pointed at the github repository. hope this helped 😃 👋
